app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -1,34 +1,46 @@
-import styled, { css } from "styled-components";
+import styled from "styled-components";
+
+const LargeH1 = styled.h1`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 72px;
+  color: inherit
+`;
```

Comment:
I'm not sure about the name. Those headers' main difference is font-size. Making one of them `largeH1` feels inconsistent. 🤔